addons/rook/template/testgrid/k8s-docker.yaml

```diff
@@ -16,6 +16,8 @@
       version: "__testver__"
       s3Override: "__testdist__"
       isBlockStorageEnabled: true
+  unsupportedOSIDs:
+    - centos-74 # Rook > 1.7 not supported on 3.10 kernel
```

Comment:
is there a preflight for this?

If there's not, can we add one?

Reply:
There isn't. I'll add one.

Reply:
Added a Rook add-on pre-init kernel check which fails if kURL detects a `3.10.0-693` kernel.